connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -3314,4 +3336,12 @@ class SparkConnectPlanner(val sessionHolder: SessionHolder) extends Logging {
       session.catalog.listCatalogs().logicalPlan
     }
   }
+
+  private def userId(): String = {
+    sessionHolder.userId
+  }
+
+  private def sessionId(): String = {
+    sessionHolder.sessionId
+  }
```

Comment:
nit: there is already a def session = sessionHolder.session on top of the file
add it there as one liners
```
private def userId = sessionHolder.userId
private def sessionId = sessionHolder.sessionId
```
by the way check it that `def session` needs to be public...

Reply:
Updated to private[connect]. I see 1 test ExamleCommandPlugin using it.